WMF Framework/Remote Notifications/Model/RemoteNotificationType.swift

```diff
@@ -169,13 +169,13 @@ public extension RemoteNotificationType {
             .pageLinked,
             .connectionWithWikidata,
             .emailFromOtherUser,
-            .mentionInTalkPage, //todo: combine this and edit summary mention to "received mention"?
```

Comment:
Already confirmed that these do not need to be combined in https://phabricator.wikimedia.org/T301129.